test/WebJobs.Script.Tests/TestScripts/Powershell/BlobTriggerToBlob-Powershell/function.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
remove the "-Powershell" suffix on this folder, since it is already scoped to PowerShell.